File: src/main/java/com/yzh/yingshi/entity/Pet.java
package com.yzh.yingshi.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("pet")
public class Pet {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long userId;
    private String petName;
    private String petType;
    private Integer age;
    private String gender;
    private String remark;
    private String avatarUrl;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: frontend/src/types/pet.ts
export interface PetVO {
  id: number;
  petName: string;
  petType: string;
  age: number;
  gender: string;
  remark: string;
  avatarUrl: string;
  createdAt: string;
}

export interface PetCreateRequest {
  petName: string;
  petType: string;
  age?: number;
  gender?: string;
  remark?: string;
  avatarUrl?: string;
}
